osquery/tables/system/windows/drivers.h

```diff
@@ -1,89 +0,0 @@
-/**
```

Comment:
Why are you removing this header altogether?

Reply:
It didn't seem needed to me. The code for all of this is relatively contained, and not sure that other tables require explicit access to accessing the drivers list outside of the table implementation.

Reply:
We should reference this in the issue as one of the breaking reasons this is marked as an API change.